docs/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md

```diff
@@ -0,0 +1,631 @@
+| `permissions` | Array of Objects | Permissions declared in the `permissions.js` file of the plugin                                                                                                                                                          |
+
+#### createSettingSection()
+
+**Type**: `Function`
+
+Create a new settings section.
+
+The function takes 2 arguments:
+
+| Argument        | Type             | Description                                                                                                                                                                                                                                                                                                                   |
+| --------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| first argument  | Object           | Section label:<ul><li>`id` (String): section id</li><li>`intlLabel` (Object): localized label for the section, following the [React Int'l](https://formatjs.io/docs/react-intl) convention, with:<ul><li>`id`: id used to insert the localized label</li><li>`defaultMessage`: default label for the section</li></ul></li></ul> |
+| second argument | Array of Objects | Links included in the section                                                                                                                                                                                                                                                                                                 |
+
+:::note
+`intlLabel.id` are ids used in translation files (`./plugins/[plugin-name]/admin/src/translations/[language.json]`)
+:::
+
+**Example:**
+
+```jsx
+// my-plugin/admin/src/index.js
+
+const myComponent = async () => {
+  const component = await import(
+    /* webpackChunkName: "users-providers-settings-page" */ './pages/Providers'
+  );
+
+  return component;
+};
+
+export default {
+  register(app) {
+    app.createSettingSection(
+	   { id: String, intlLabel: { id: String, defaultMessage: String }, // Section to create
+		 [ // links
+			 {
+			   intLabel: { id: String, defaultMessage: String },
+			   id: String,
+			   to: String,
+			   Component: myComponent,
+			   permissions: Object[]
+		 ] 
+  }
+}
+```
+
+#### addSettingLink()
+
+**Type**: `Function`
+
+Add a unique link to an existing settings section.
+
+**Example:**
+
+```jsx
+// my-plugin/admin/src/index.js
+
+export default {
+  bootstrap(app) {
+		// Adding a single link
+		app.addSettingLink(
```

Comment:
Hey @pwizla , thanks for the awesome docs :).  I think this should be `addSettingsLink` with an `s` on settings.  

<img width="545" alt="Screenshot 2021-10-05 at 16 32 15" src="https://user-images.githubusercontent.com/26598053/136098533-a32d6012-0603-4329-a46d-336e69f492bd.png">